artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyConnector.java

```diff
@@ -528,38 +534,54 @@ public synchronized void start() {
          return;
       }
 
-      if (remotingThreads == -1) {
+      boolean defaultRemotingThreads = remotingThreads == -1;
+
+      if (defaultRemotingThreads) {
          // Default to number of cores * 3
          remotingThreads = Runtime.getRuntime().availableProcessors() * 3;
       }
 
       String connectorType;
 
-      if (useEpoll && CheckDependencies.isEpollAvailable()) {
+      if (useIoUring && CheckDependencies.isIoUringAvailable()) {
+         //IO_URING should default to 1 remotingThread unless specified in config
```

Comment:
It should use less than we use too with epoll - but 1 could be quite low.

And check  https://github.com/netty/netty/pull/15524 if can help to scale up the number of event loops if required